modules/tinymce/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## 5.8.0 - 2021-05-06
 
 ### Added
+- Added a new `toolbar_sticky_offset` setting to allow the toolbar to be offset from the top of the page #TINY-7337
```

Comment:
Sorry we've let this stall @camiloHimura 😞 Anyways, this changelog was added to the incorrect location originally so could you please merge in develop and move this to the 5.9/Unreleased section?

- [ ] Move changelog entry

Reply:
Updated